msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -68,28 +69,29 @@ The following steps help you to enable SSO in your application:
 > These changes are based on the templates we scaffold.
 
 ---
+
 <br>
 <br><details>
 <summary><b>Tab project
 </b></summary>
 
-1. Copy `auth-start.html` and `auth-end.htm`** in `auth/public` folder to `tabs/public/`. Teams Toolkit registers these two endpoints in Azure AD for Azure AD's redirect flow.
+1. Copy `auth-start.html` and `auth-end.htm`\*\* in `auth/public` folder to `tabs/public/`. Teams Toolkit registers these two endpoints in Azure AD for Azure AD's redirect flow.
```

Comment:
As per guidelines to write a folder path: In command syntax, use path to represent only the folder portion of the full path: copy [drive:][path]filename.

Request you to check and update the file path names.